src/vs/workbench/services/configurationResolver/common/configurationResolverSchema.ts

```diff
@@ -73,9 +73,28 @@ export const inputsSchema: IJSONSchema = {
 							},
 							options: {
 								type: 'array',
-								description: nls.localize('JsonSchema.input.options', 'An array of strings that defines the options for a quick pick.'),
+								description: nls.localize('JsonSchema.input.options', "An array of strings that defines the options for a quick pick."),
 								items: {
-									type: 'string'
+									oneOf: [
+										{
+											type: 'string'
+										},
+										{
+											type: 'object',
+											required: ['value'],
+											additionalProperties: false,
+											properties: {
+												name: {
```

Comment:
Can this be `label` instead of `name`? In Tasks, we call it label, and in general we try to use label to indicate that it is a string that will be shown in the UI.